README.md

```diff
@@ -45,16 +45,18 @@ Alternatively you can download one of the pre-built binaries for the supported p
 
 These builds are generated once a day from `master` branch starting at 01:00 GMT
 
-In order to get the latest nightly build for your platform use the following links replacing `<DATE>` with the current
-date, using the format YYYYMMDD (i.e for 2019/Aug/06 use `20190806` )
-
-- **Linux 64 bit**: `https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-<DATE>_Linux_64bit.tar.gz`
-- **Linux 32 bit**: `https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-<DATE>_Linux_32bit.tar.gz`
-- **Linux ARM 64 bit**: `https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-<DATE>_Linux_ARM64.tar.gz`
-- **Linux ARM 32 bit**: `https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-<DATE>_Linux_ARMv7.tar.gz`
-- **Windows 64 bit**: `https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-<DATE>_Windows_64bit.zip`
-- **Windows 32 bit**: `https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-<DATE>_Windows_32bit.zip`
-- **Mac OSX**: `https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-<DATE>_macOS_64bit.tar.gz`
+In order to get the latest nightly build for your platform use the following links: 
+
+- **Linux 64 bit**: https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_Linux_64bit.tar.gz
+- **Linux 32 bit**: https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_Linux_32bit.tar.gz
+- **Linux ARM 64 bit**: https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_Linux_ARM64.tar.gz
+- **Linux ARM 32 bit**: https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_Linux_ARMv7.tar.gz
+- **Windows 64 bit**: https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_Windows_64bit.zip
+- **Windows 32 bit**: https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_Windows_32bit.zip
+- **Mac OSX**: https://downloads.arduino.cc/arduino-cli/nightly/arduino-cli_nightly-latest_macOS_64bit.tar.gz
+
+These links redirect to latest generated builds replacing `latest` with the latest available build
+date, using the format YYYYMMDD (i.e for 2019/Aug/06 `latest` is replaced with `20190806` )
```

Comment:
Not sure we need to disclose the strategy used, WDYT?

Reply:
Good point, 
I was thinking that if you are using the latest link inside some automation, would be good to know that you will receive a redirect. This way you can configure your automation to follow it.
Does this makes sense to you?

Reply:
Yup it does, in this case should we mention the URL will always respond with 302?

Reply:
:+1: see 35ac883

Reply:
:cake: